includes/settings/functions.php

```diff
@@ -503,6 +503,39 @@ function admin_welcome_notice() {
 	<?php
 }
 
+function admin_no_payment_pointer_notice() {
```

Comment:
This function should at the very least have a PHPDoc comment explaining what it does.